doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+    "$$\n",
+    "\n",
+    "we can categorise drift into a number of types:\n",
+    "\n",
+    "- **Covariate drift**: Also referred to as input drift, this occurs when $P(\\mathbf{Y}|\\mathbf{X})$ = $P_{ref}(\\mathbf{Y}|\\mathbf{X})$, but the distribution of the input data has shifted $P(\\mathbf{X}) \\ne P_{ref}(\\mathbf{X})$. In a mortgage risk classification task, an example of this would be if at inference, applications for properties with more bedrooms than the properties present in the training data were considered. The model may now be extrapolating, potentially resulting in unreliable predictions. \n",
+    "\n",
+    "- **Prior drift**: Also referred to as label drift, this occurs when $P(\\mathbf{X}|\\mathbf{Y})=P_{ref}(\\mathbf{X}|\\mathbf{Y})$, but the distribution of the outputs has shifted $P(\\mathbf{Y}) \\ne P_{ref}(\\mathbf{Y})$. Using the same example again, an increase in the proportion of applications classsed as \"approved\" would be an example of prior drift. The model could still be accurate, but accuracy metrics might be affected. \n",
+    "\n",
+    "- **Concept drift**: This occurs when the process generating $y$ from $x$ has changed, such that $P(\\mathbf{Y}|\\mathbf{X}) \\ne P_{ref}(\\mathbf{Y}|\\mathbf{X})$. An example of this could be if a change in the wider economic climate resulted in a change in the risk for a given mortgage application. It is possible that the model might no longer give a suitable approximation of the true process. \n",
+    "\n",
+    "Below, these different types of drift are visualised for a simple two-dimensional classification problem. It is possible for a drift to fall under more than one category, for example the *prior drift* below also happens to be a case of *covariate drift*. \n",
+    "\n",
+    "![2d drift example](images/bg_2d_drift.png) \n",
+    "\n",
+    "It is relatively easy to spot drift by eyeballing these figures here. However, the task becomes considerably harder for high-dimensional real problems, especially since real-time ground truths are not typically available. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Detecting drift <a class=\"anchor\" id=\"drift_detection\"></a>\n",
+    "\n",
+    "[Alibi Detect](https://github.com/SeldonIO/alibi-detect) offers a wide array of methods for detecting drift (see [here](../overview/algorithms.md)), many of which are examined in the NeurIPS 2019 paper [Failing Loudly: An Empirical Study of Methods for Detecting Dataset Shift](https://arxiv.org/abs/1810.11953). Generally, these aim to determine whether the distribution $P(\\mathbf{x})$ has drifted from a reference distribution $P_{ref}(\\mathbf{x})$, where $\\mathbf{x}$ may represent input data $\\mathbf{X}$, true output data $\\mathbf{Y}$, or some form of model output, depending on what type of drift we wish to detect. \n",
```

Comment:
"many of which" -> "some of which" :)
We've implemented and tested much more than what is talked about in that paper.